docs/ratify-configuration.mdx

```diff
@@ -3,47 +3,710 @@ title: Configuration
+**Configuration Methods**: Ratify can be configured using either mounted `config.json` files or Kubernetes Custom Resources (CRDs). Since `config.json` configurations can be converted to CRDs and CRDs provide better Kubernetes integration, this document uses CRD examples throughout the tutorial.
 
-## CRDs
-Ratify also supports configuration through K8s [CRDs](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/). The configuration can be updated using natively supported `kubectl` commands.
+## Table of Contents
 
-When running Ratify in a pod, the `ConfigMap` will be mounted in the pod at the default configuration file path. Ratify will initialize with specifications from the configuration file. CRDs will override store and verifier defined in the configuration file if they exist at runtime. Our team is in the process of converting configuration components into Ratify CRDs to support a more native k8s experience. Please review ratify CRDs samples [here](https://github.com/notaryproject/ratify/tree/main/config/samples).
+- [Overview](#overview)
+- [Executor Configuration Structure](#executor-configuration-structure)
+- [Verifier Configuration](#verifier-configuration)
+  - [Notation Verifier](#notation-verifier)
+  - [Cosign Verifier](#cosign-verifier)
+- [Key Provider Configuration](#key-provider-configuration)
+  - [Inline Key Provider](#inline-key-provider)
+  - [Filesystem Key Provider](#filesystem-key-provider)
+  - [Azure Key Vault Provider](#azure-key-vault-provider)
+- [Store Configuration](#store-configuration)
+  - [Registry Store](#registry-store)
+  - [Filesystem OCI Store](#filesystem-oci-store)
+- [Policy Enforcer Configuration](#policy-enforcer-configuration)
+  - [Threshold Policy](#threshold-policy)
+- [Complete Configuration Examples](#complete-configuration-examples)
+- [Additional Resources](#additional-resources)
 
-Currently supported components through CRDs are:
+## Overview
 
-- [Verifiers](./reference/custom%20resources/verifiers.md)
-- [Stores](./reference/custom%20resources/stores.md)
-- [Key Management Providers](./reference/custom%20resources/key-management-providers.md)
-- [Policy Provider](./reference/custom%20resources/policies.md)
+Ratify executors provide a scoped approach to artifact verification, allowing different verification policies and configurations for different registries, repositories, or artifact types. Each executor contains:
 
-### Get CRDs
-Our helms charts are wired up to initialize CRs based on chart values. 
-After Ratify installation, you can use the `kubectl` command to review the currently active configuration.
+- **Scopes**: Define which registries/repositories this executor handles
+- **Verifiers**: Configure signature verification (Notation, Cosign, etc.)
+- **Stores**: Define how to access artifacts
```

Comment:
```suggestion
- **Stores**: Define how to access artifacts in OCI registries or OCI image layout
```

Reply:
Would it bring more clarity if explain "artifacts in OCI registries or OCI image layout"?